include/tvm/relax/attrs/nn.h

```diff
@@ -29,6 +29,49 @@
 namespace tvm {
 namespace relax {
 
+/*! \brief Attributes used in Conv1d operator */
+struct Conv1DAttrs : public tvm::AttrsNode<Conv1DAttrs> {
```

Comment:
looks it is quite similar to Conv2Dattrs, probably we can unify them into a ConvAttrs

Reply:
It seems possible to unify them together. We can do sanity checks in respective struct info inference of conv1d and conv2d.

Reply:
Let us keep them separate for now